nicegui/static/nicegui.js

```diff
@@ -142,16 +176,6 @@ function renderRecursively(elements, id) {
     style: Object.entries(element.style).reduce((str, [p, val]) => `${str}${p}:${val};`, "") || undefined,
     ...element.props,
   };
-  Object.entries(props).forEach(([key, value]) => {
-    if (key.startsWith(":")) {
-      try {
-        props[key.substring(1)] = eval(value);
-        delete props[key];
-      } catch (e) {
-        console.error(`Error while converting ${key} attribute to function:`, e);
-      }
-    }
-  });
```

Comment:
@jdoiro3 - Revert this change.

Reply:
Resolved in https://github.com/zauberzeug/nicegui/pull/3107/commits/30ec33c8f059651137a5cb8562f28a9c877014e4.